### PR TITLE
Ux

### PR DESCRIPTION
**Pull request by Ruzeb**
- Added react Icons to serve as buttons to close all of the modals and the describe block in react
- converted all of the react modals to be draggable for user experience

### CODE REVIEW

File: src/components/Modals/Modal.module.scss [hunk 1]
```diff
@@ -130,7 +138,7 @@
     -webkit-appearance: none;
     -moz-appearance: none;
   }
-  button:focus {
+  button:hover {
```
[the remark this answers is not in the document]
Comment (Ruzeb) [reply src/components/Modals/Modal.module.scss:142]: fixed

File: src/components/Modals/Modal.jsx [hunk 2]
```diff
@@ -15,6 +15,11 @@ import AccordionDetails from '@material-ui/core/AccordionDetails';
 import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
 import cn from 'classnames';
 import { GlobalContext } from '../../context/reducers/globalReducer';
+import { DragDropContext, Droppable, DropResult } from 'react-beautiful-dnd';
```
Comment (huyqbui) [inline src/components/Modals/Modal.jsx:18]: let's remove this since we're not using it
```suggestion
```

**Review by huyqbui — CHANGES_REQUESTED** [review]
Just needs removing the unused import statement, but everything else looks good!

File: src/components/Modals/Modal.jsx (hunk 2 above)
Comment (Ruzeb) [reply src/components/Modals/Modal.jsx]: resolved

**Review by huyqbui — APPROVED** [review]

**Pull request merged**